d2core/d2map/d2mapgen/act1_overworld.go

```diff
@@ -284,7 +284,7 @@ func (g *MapGenerator) generateWilderness1Contents(rect d2geom.Rectangle) {
 		for x := 0; x < rect.Width; x++ {
 			tile := g.engine.Tile(rect.Left+x, rect.Top+y)
 			tile.RegionType = d2enum.RegionIdType(levelDetails.LevelType)
-			tile.Components.Floors = []d2ds1.FloorShadowRecord{{Prop1: 1, Style: 0, Sequence: 0}} // wildernessGrass
+			tile.Components.Floors = []d2ds1.FloorShadow{{Prop1: 1, Style: 0, Sequence: 0}} // wildernessGrass
```

Comment:
`d2ds1.Floor` and `d2ds1.Shadow` are both aliases of `d2ds1.FloorShadow`, so where it makes more sense to just refer to something as a `Floor` or a `Shadow` we can now do so.